Activity.js

```diff
@@ -9,10 +9,48 @@ class Activity {
   }
 
   startTimer() {
-
+      var currentTime = Date.parse(new Date());
+      var nye = new Date(currentTime + parseInt(minutesInput.value)*60*1000 + parseInt(secondsInput.value)*1000);
+      
+      var second = 1000;
+      var minute = second * 60;
+      var hour = minute * 60;
+      var day = hour * 24;
+      var timer;
+      
```

Comment:
There's some refactoring to do here, as we discussed on the group call. Will tackle tomorrow.